testutil/compose/fuzz/fuzz_test.go

```diff
@@ -58,3 +58,35 @@ func TestBeaconFuzz(t *testing.T) {
 	err = compose.Auto(context.Background(), autoConfig)
 	testutil.RequireNoError(t, err)
 }
+
+func TestP2PFuzz(t *testing.T) {
+	if !*fuzz {
+		t.Skip("Skipping p2p fuzz integration test")
+	}
+
+	dir, err := os.MkdirTemp("", "")
+	require.NoError(t, err)
+
+	conf := compose.NewDefaultConfig()
+	conf.DisableMonitoringPorts = true
+	conf.BuildLocal = true
+	conf.P2PFuzz = true
```

Comment:
seems like all nodes are fuzzing so all duties will fail.

Reply:
this config field is introduced to enable p2p fuzz in "one" of the nodes in compose cluster.